src/chains/ethereum_1/ethTransaction.ts

```diff
@@ -111,9 +111,11 @@ export class EthereumTransaction implements Transaction {
     const { nonce = null } = this._options || {}
     let { gasPrice = null, gasLimit = null } = this._options || {}
     const { to, value, data } = this._actionHelper.raw
-    // 1 * ... is the gasPrice multiplayer currently hardcoded, ready to be replaced by an optional parameter
-    gasPrice = isNullOrEmpty(gasPrice) ? 1 * (await this._chainState.getGasPrice()) : gasPrice
+    // 0.000000001 * ... is the gasPrice multiplayer currently hardcoded, ready to be replaced by an optional parameter
+    // Convert gas price returned from getGasPrice to Gwei
+    gasPrice = isNullOrEmpty(gasPrice) ? 0.000000001 * (await this._chainState.getGasPrice()) : gasPrice
```

Comment:
If gasPrice is expressed in some eth unit then you should use the ethUnit converter in chains to convert the price instead of multiplying it to a constant

Reply:
there are 2 functions currently in chainjs:
toEthUnit: just returns unit as EthUnit enum value. Doesn't do any value conversion
toWei: uses web3's exposed method(utils.toWei) to convert to wei

There is no existing function to convert to Gwei
@traylewin

Reply:
I can add a function to convert from wei to Gwei if that makes sense

Reply:
OK sure. toGweiFromWei makes sense

Reply:
ok doing the change now. Would like to publish today